website/src/components/AmbassadorCard/index.tsx

```diff
@@ -0,0 +1,33 @@
+import LearnCard from "../LearnCard";
+
+interface AmbassadorCardProps {
+  title: string;
+  role: string;
+  company: string;
+  companyLink: string;
+  personalLink: string;
+  img: string;
+}
+
+export const createAmbassadorCard = ({
```

Comment:
Conventionally, functional components are still named in PascalCase
```suggestion
export const AmbassadorCard = ({
```

Reply:
Good to know! TY!